DOCSP-22867: describe ChangeStreamDocument (#406)
mongodb/docs-java: source/fundamentals/crud/read-operations/change-streams.txt

Example
~~~~~~~

The following code example shows how to open a change stream on a
collection and print change stream events whenever the data in the
collection changes:

.. literalinclude:: /includes/fundamentals/code-snippets/change-streams/ChangeStreams.java

>    :language: java

Author: The way the code is set up right now, a collection is passed as a parameter. I'm going to edit this to instantiate the collection inside the function though.

      ...
   }

> .. note:: ChangeStreamIterable and ChangeStreamDocument Types

Author: Agreed - but what do you mean about changing the introduction?

> .. note:: ChangeStreamIterable and ChangeStreamDocument Types

Reviewer: Oh - I just meant the wording would need to be reworked. It's all good!

   
   The driver stores change stream events in a variable of type

>    ``ChangeStreamIterable``. In the preceding example, the driver stored
>    individual change stream events as ``ChangeStreamDocument`` objects,
>    since we specified that the driver should populate the ``ChangeStreamIterable``
>    with ``Document`` types.

Reviewer: [Suggestion]

A couple of things here. First, the sentence is a little long, so I would recommend breaking it up. Also, I would put the action that we did first, to put the focus on that, and then have the effect later. Something like the following:

```suggestion
   ``ChangeStreamIterable``. In the preceding example, we specified the driver should
   populate the ``ChangeStreamIterable`` with ``Document`` types. As a result, the driver
   stores the individual change stream events as ``ChangeStreamDocument`` objects.
```
